libs/components/core/src/lib/modules/overlay/overlay.component.ts

```diff
@@ -176,17 +176,16 @@ export class SkyOverlayComponent implements OnInit, OnDestroy {
 
     this.targetRef.clear();
 
-    const factory = this.#resolver.resolveComponentFactory(component);
     const injector = Injector.create({
       providers,
       parent: this.#injector,
     });
 
-    const componentRef = this.targetRef.createComponent<C>(
-      factory,
-      undefined,
-      injector
-    );
+    const componentRef = createComponent<C>(component, {
```

Comment:
Any reason we can't use the `targetRef`'s `createComponent` method instead? Seems like a simpler implementation:
```
const componentRef = this.targetRef.createComponent(component, {
  injector,
});
```

Reply:
Good call. That's easier.